components/brave_wallet_ui/options/asset-options.ts

```diff
@@ -178,13 +195,17 @@ export const NewAssetOptions: BraveWallet.BlockchainToken[] = [
     decimals: 0,
     visible: true,
     tokenId: '0x42a5',
-    coingeckoId: ''
+    coingeckoId: '',
+    chainId: '0x1'
   }
 ]
 
 // Use only with storybook as dummy data.
 export const AccountAssetOptions: BraveWallet.BlockchainToken[] = [
```

Comment:
Let's move this to a mocks of fixtures folder, since this is dummy data

Reply:
`asset-options.ts` is used throughout the wallet project and would require a refactor of things unrelated to this PR.
Best to keep things separate, if you would like to have this moved, please create an issue to refactor.